_news-items/2023-08-30-panelists.html

```diff
@@ -8,7 +8,7 @@
 <p>
   We are very fortunate to welcome five industry leaders to participate at the
   <a href="../Foundation/Summit/2023">seL4 Summit 2023</a>, in a session <a
-  href="https://events.linuxfoundation.org/sel4-summit/program/schedule/">OS on
+  href="../Foundation/Summit/2023/abstracts2023.html#a-panel">OS on
```

Comment:
This will work here, so no need to change, but in general we should avoid relative links in news items, because the news item will be rendered in multiple different locations (landing page, menus, news/, news/<explicit year>).

So we'd need to do the `{{ '/Foundation/etcl' | relative_url }}` dance or use `{% link Foundation/etc %}`

Reply:
A lot of news items have this issue. Should I go through them all to replace the relative links?

Reply:
It's only relevant for the ones in 2025 -- once they are in the older years they will only be accessible from that year's page, so they should be fine. But for 2025 that would be good to do, yes.

Reply:
I fixed them as part of this PR, since there were only two news items in 2025 that required relative links. (And it was part of the discussion here.)